cmd/mayactl/app/command/snapshot/revert.go

```diff
@@ -24,25 +24,37 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	snapshotrevertHelpText = `
+This command rolls back the volume data to the specified snapshot. Once the roll back 
```

Comment:
... back volume data to a specified snapshot.

Reply:
done.